lib/chef/chef_fs/data_handler/acl_data_handler.rb

```diff
@@ -11,8 +11,8 @@ def normalize(acl, entry)
             'read' => {},
             'update' => {},
             'delete' => {},
-            'grant' => {}
-            })
+            'grant' => {},
+            },)
```

Comment:
the `{}` in the function call are actually optional here as well...  so all these get cleaned up later...